sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Custom/Models/BackupGenericProtectedItem.cs

```diff
@@ -20,5 +21,13 @@ public int? SoftDeleteRetentionPeriod
                 SoftDeleteRetentionPeriodInDays = value;
             }
         }
+
+        /// <summary> ID of the backup policy with which this item is backed up. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ResourceIdentifier PolicyId
+        {
+            get => string.IsNullOrEmpty(PolicyStringId) ? null : new ResourceIdentifier(PolicyId);
+            set => PolicyId = new ResourceIdentifier(PolicyStringId);
```

Comment:
this is not correct. `value` is not used here therefore when people assign something to this property, nothing will happen.
We should do:
```suggestion
            set => PolicyStringId = value.ToString();
```

Reply:
fixed.